VeroliBGV/BGV008.xml

```diff
@@ -0,0 +1,836 @@
+                              </item>
+                              <item xml:id="q14" n="13">
+                                 <dim unit="leaf">8</dim>
+                                 <locus from="97r" to="104v"/>
+                              </item>
+                              <item xml:id="q15" n="14">
+                                 <dim unit="leaf">8</dim>
+                                 <locus from="105r" to="112v"/>
+                              </item>
+                              <item xml:id="q16" n="15">
+                                 <dim unit="leaf">8</dim>
+                                 <locus from="113r" to="120v"/>
+                              </item>
+                              <item xml:id="q17" n="16">
+                                 <dim unit="leaf">8</dim>
+                                 <locus from="121r" to="128v"/>
+                              </item>
+                              <item xml:id="q18" n="17">
+                                 <dim unit="leaf">8</dim>
+                                 <locus from="129r" to="136v"/>
+                              </item>
+                              <item xml:id="q19" n="18">
+                                 <dim unit="leaf">10</dim>
+                                 <locus from="137r" to="146v"/>
+                              </item>
+                              <item xml:id="q20" n="19">
+                                 <dim unit="leaf">8</dim>
+                                 <locus from="147r" to="154v"/>
+                              </item>
+                              <item xml:id="q21" n="20">
+                                 <dim unit="leaf">8</dim>
+                                 <locus from="155r" to="162v"/>
+                              </item>
+                              <item xml:id="q22" n="21">
+                                 <dim unit="leaf">8</dim>
+                                 <locus from="163r" to="170v"/>
+                              </item>
+                              <item xml:id="q23" n="22">
+                                 <dim unit="leaf">6</dim>
+                                 <locus from="171r" to="176v"/>
+                              </item>                      
+                           </list>
+                        </collation>
+                        <condition key="good">
+                           The back board is cracked and repaired with three threads.
+                           The sewing has been repaired in correspondence of the center fold of <ref target="#q1 #q23"/>. 
+                           The manuscript is damaged by humidity: water stains are found especially towards the end of the manuscript (<locus from="171" to="176"/>).
+                           Some folios are stained with dirt, especially in corrispondence of the centerfold of several quires (e.g. <locus target="#30v #31r #85r"/>).
```

Comment:
correspondence
The expression "in correspondence" is correct?

Reply:
Yes, "in correspondence of" is correct